spec/controllers/ops_controller_spec.rb

```diff
@@ -42,7 +42,7 @@
                                      :active_tab  => 'db_connections',
                                      :trees       => {:vmdb_tree => {:active_node => 'root'}}}}
     session[:settings] = {:views => {}, :perpage => {:list => 10}}
-    expect(controller).to receive(:render)
+    expect(controller).to receive(:head)
```

Comment:
Finally found the problem: this spec has been dead in the water since 3f75dc60720cb88e7c1402aa317228d92e447c56 broke it -- it no longer actually tests what it claims to be testing.

Instead of working out why 3f75dc60720cb88e7c1402aa317228d92e447c56 actually made it fail, and what should happen to make it actually-useful again, I've just re-hacked it to match the changed implementation in the error handler. :see_no_evil: